js/app.js

```diff
@@ -108,59 +72,125 @@ function pickRandomNum(){
     randomNum = Math.floor(Math.random() * stockImages.length);
     compare(randomNum);
   }
+  console.log(currNum);
 }
+
 //assigning currNum to prevNum
 function tranCurrToPrev(){
   for(i = 0; i < 3; i++){
     prevNum[i] = currNum[i];
   }
-  currNum = [];
+  if(i === 3){
+    currNum = [];
+  }
+
 }
 
 //set filepathg of image1, image2, and image3
 function setImgFilepath(){
   imgEl1.src = objStore[prevNum[0]].filepath;
+  imgEl1.alt = objStore[prevNum[0]].name;
   imgEl2.src = objStore[prevNum[1]].filepath;
+  imgEl2.alt = objStore[prevNum[1]].name;
   imgEl3.src = objStore[prevNum[2]].filepath;
+  imgEl3.alt = objStore[prevNum[2]].name;
+}
+
+for(i = 0; i < 3; i++){
+  pickInitialNum();
 }
```

Comment:
You do not need to do this. If you just call `pickRandomNum` it will do the exact same thing. Also, if you just use  `pickRandomNum` the `pickInitialNum` function will be obsolete and unnecessary code.